examples/mobile/HomeApp/index.html

```diff
@@ -37,8 +37,10 @@
 			<div class="ui-content-subheader">
 				Weather Forecast
 			</div>
-			<div class="ui-content-area">
-				<a style="border-radius: 26px" class="weatherwidget-io" href="https://forecast7.com/en/37d57126d98/seoul/" data-label_1="Seoul" data-label_2="South Korea" data-theme="pure" >Seoul South Korea</a>
+			<div class="ui-content-area app-weather-container">
+				<a style="border-radius: 26px" class="weatherwidget-io" href="https://forecast7.com/en/37d57126d98/seoul/" data-label_1="Seoul" data-label_2="South Korea" data-theme="pure" >
+					<div class="ui-progress" data-type="indeterminatecircle" data-size="large"></div>
```

Comment:
Patch looks good but in my opinion the progress is too close the edge of content area.

Reply:
Done!